### PR TITLE
docs: add deploy cluster profile updates tutorial DOC-921

### PR DESCRIPTION
**Pull request by addetz**
## Describe the Change

This patch creates the "Deploy Cluster Profile Updates" tutorial.

## Review Changes

💻 [Preview URL](https://deploy-preview-1912--docs-spectrocloud.netlify.app/clusters/cluster-management/update-k8s-cluster)

🎫 [Jira Ticket](https://spectrocloud.atlassian.net/browse/DOC-921)

### CODE REVIEW

File: docs/docs-content/clusters/cluster-management/update-k8s-cluster.md [hunk 1]
```diff
@@ -0,0 +1,920 @@
...
+
+The plan makes minor updates to multiple resources. It also updates the cluster in place to use the configured cluster profile version. The `kubeconfig` file is replaced as well.
+
+To deploy all the resources, use the `apply` command.
+
+```shell
+terraform apply -auto-approve
+```
+
+Log in to [Palette](https://console.spectrocloud.com).
```
Comment (karl-cardenas-coding) [inline docs/docs-content/clusters/cluster-management/update-k8s-cluster.md:746]: Did we log out? I think we can just say "Navigate back to the Palette UI"

File: docs/docs-content/clusters/cluster-management/update-k8s-cluster.md [hunk 2]
```diff
@@ -0,0 +1,920 @@
...
+
+The plan adds the cluster profile resource that you uncommented and changes the host cluster to use it. It also makes other minor updates, as well as recreates the local **kubeconfig** file.
+
+To deploy all the resources, use the `apply` command.
+
+```shell
+terraform apply -auto-approve
+```
+
+Log in to [Palette](https://console.spectrocloud.com).
```
Comment (karl-cardenas-coding) [inline docs/docs-content/clusters/cluster-management/update-k8s-cluster.md:679]: Log in? Maybe jump back to the Palette tab in the browser

File: docs/docs-content/clusters/cluster-management/update-k8s-cluster.md [hunk 3]
```diff
@@ -0,0 +1,920 @@
...
+Set the value of `replicas` to `3` in the right-hand dialog. This removes your cluster override. Click on **Confirm updates** once you have finished reviewing your changes.
+
+Palette updates your cluster according to cluster profile specifications. Once these changes are complete, select the **Workloads** tab. Then, select the **hello-universe** namespace.
+
+Three replicas of the **hello-universe-deployment** are available. The cluster profile update is now reflected by your cluster.
+
+</TabItem>
+<TabItem label="Terraform workflow" value="Terraform">
+
+Log in to [Palette](https://console.spectrocloud.com).
```
Comment (karl-cardenas-coding) [inline docs/docs-content/clusters/cluster-management/update-k8s-cluster.md:802]: Same feedback as above

**Review by karl-cardenas-coding — APPROVED** [review]
Great tutorial and good job explaining the concepts. I left some minor comments and suggestions below.

File: docs/docs-content/clusters/cluster-management/update-k8s-cluster.md [hunk 4]
```diff
@@ -0,0 +1,920 @@
...
+
+Next, issue the `plan` command to preview the changes.
+
+```shell
+terraform plan
+```
+
+Output:
+```shell
+Plan: 6 to add, 0 to change, 0 to destroy.
```
[the remark this answers is not in the document]
Comment (addetz) [reply docs/docs-content/clusters/cluster-management/update-k8s-cluster.md:227]: Explained that this is for Azure only. I added separate expected output for the other cloud providers right under.

File: docs/docs-content/clusters/cluster-management/update-k8s-cluster.md [hunk 5]
```diff
@@ -0,0 +1,920 @@
...
+```
+
+If you change the desired cloud provider's toggle variable to `true,` you will receive an output message that the following six new resources are planned.
```
[the remark this answers is not in the document]
Comment (addetz) [reply docs/docs-content/clusters/cluster-management/update-k8s-cluster.md]: Removed the count here.

File: docs/docs-content/clusters/cluster-management/update-k8s-cluster.md [hunk 6]
```diff
@@ -0,0 +1,920 @@
...
+- A cluster profile and a host cluster for the [*hello-universe*](https://github.com/spectrocloud/hello-universe) application.
+- A cluster profile and a host cluster for the [*hello-universe-api*](https://github.com/spectrocloud/hello-universe-api) application.
+- The [*kubeconfig*](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/) file of the provisioned *hello-universe-api* cluster. This is a local file that is created in your current working directory. You will need this file later, so do not to delete it.
+- An SSH key required for deployment to Azure. This resource is not created if you choose to deploy to another cloud provider.
```
[the remark this answers is not in the document]
Comment (addetz) [reply docs/docs-content/clusters/cluster-management/update-k8s-cluster.md:239]: I think the confusion will be addressed with the output for the other providers.

File: docs/docs-content/clusters/cluster-management/update-k8s-cluster.md [hunk 7]
```diff
@@ -0,0 +1,920 @@
...
+          - name: API_URI
+            value: ${api_uri}
+```
+
+The manifest deploys the [*hello-universe*](https://github.com/spectrocloud/hello-universe) application with the extra environment variable `API_URI`. This environment variable allows you to specify a hostname and port for the *hello-universe* API server. Check out the [*hello-universe* readme](https://github.com/spectrocloud/hello-universe?tab=readme-ov-file#connecting-to-api-server) to learn more about how to expand the capabilities of the *hello-universe* application with an API Server.
+
+Issue the following command to find the external IP address of the `hello-universe-api-service` that you deployed in the [Set Up Clusters](#set-up-clusters) section. Replace the placeholder `REPLACE_ME` with the cloud provider that you have chosen to deploy your clusters to - `aws`, `azure` or `gcp`. 
+
+```shell
+export KUBECONFIG=$(pwd)/REPLACE_ME-cluster-api.kubeconfig && kubectl get service hello-universe-api-service --namespace hello-universe-api --output jsonpath='{.status.loadBalancer.ingress[0].ip}'
```
[the remark this answers is not in the document]
Comment (addetz) [reply docs/docs-content/clusters/cluster-management/update-k8s-cluster.md]: Fixed this command in the tutorials PR and checked what it is on GCP. Instructed the learner to make a note of it when they issue the `terraform apply` and explained that this is what it looks like for Azure instead.

**Pull request merged**